projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+* ❌ Not suitable when the organizers of the events created through your application are unpredictable in advance
+
+#### Example
+
+In the example below, all the events created through the integration have the same organizer, "Natuurpunt Holsbeek". First you'll need to find the identifier of the organizer `Natuurpunt Holsbeek` in UiTdatabank.
+
+**request**
+
+Perform a `GET` request to the UiTdatabank API including the name of the organizer:
+
+```
+GET /organizers/?q=name.\*:"Natuurpunt Holsbeek"
+```
+
+<!-- theme: warning -->
+
+> For organizers we advice to only search for the `name` and to not include a search on `postalCode` (since this is an optional field for organizers).
+
+**response**
+
+The response now contains the identifier of "Natuurpunt Holsbeek":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116",
+         "@type":"Organizer"
+      }
+   ]
+}
+```
+
+You can now store the identifier of the organizer "Natuurpunt Holsbeek" (`e83edb7f-6f23-47cd-a730-66fd7687f116`) in your application to reuse it as organizer ID in the `POST` request of the event(s) that you want to create.
+
+### Searching existing organizers within the entry form
+
+In this solution you directly integrate with the UiTdatabank Search API in your own entry form.
+
+* ✅ Suitable when the organizers for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same organizer
+* ❌ Not recommended when the events created through your application can have a different organizer, but (1) the amount of organizers is limited & (2) the list of organizers is static and won't change a lot over time
+
+#### Example
+
+In the example below, a user in your entry form types the name "Natuurpunt Holsbeek" in the input field for the organizer. Under the hood an API call to the UiTdatabank Search API is made to check if that organizer already exists:
+
+**request**
+
+```
+GET /organizers/?q=name.\*:"Natuurpunt Holsbeek"
+```
+
+**response**
+
+The response now contains the identifier of "Natuurpunt Holsbeek":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116",
+         "@type":"Organizer"
+      }
+   ]
+}
+```
+
+If the organizer already exists (like in the example above), it is suggested in the user interface in a human-readable way so that the user can select the organizer and thus choose to reuse the organizer for the creation of his or her event.
```

Comment:
```suggestion
If one or more organizers already exist with this name (like in the example above), they are displayed as human-readable suggestions in the user interface so that the user can choose to reuse one of these organizers for the creation of his or her event.
```